bodo/pandas/frame.py

```diff
@@ -60,6 +62,23 @@
 )
 
 
+class BodoDataFrameLocIndexer(_LocIndexer):
+    def __init__(self, name, obj):
+        self.df = obj
+        super().__init__(name, obj)
+
+    def __getitem__(self, key):
+        if isinstance(key, tuple) and len(key) == 2:
```

Comment:
Add a fallback warning if not supported?

Reply:
Done.